packages/api/endpoints/orca.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const mapKeys = require('lodash/mapKeys');
+const router = require('express-promise-router')();
+
+const { lambda } = require('@cumulus/aws-client/services');
+
+const mapKeysToOrca = {
+  granuleId: 'granule_id',
+};
+
+/**
+ * List recovery request status
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function listRequests(req, res) {
+  const query = req.query || {};
```

Comment:
And if no query is provided, what do we expect to get back? Did we talk about this option with them? I think we said it would always at least include a jobId

Reply:
no query parameter will return all the requests I think.  It depends how the request_status lambda is implemented.  Will can we get a jobId?